XamarinCommunityToolkit/Behaviors/Animations/AnimationBehaviour.shared.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System.Linq;
+using Xamarin.Forms;
+
+namespace XamarinCommunityToolkit.Behaviors.Animations
+{
+    public class AnimationBehaviour : EventToCommandBehavior
```

Comment:
While going through this again I noticed this one is called Behavio*u*r. The rest doesn't have the u so please rename :)

Reply:
Totally my bad! A big typo :P